bin/pretty-quick.js

```diff
@@ -7,7 +7,11 @@ const mri = require('mri');
 
 const prettyQuick = require('..').default;
 
-const args = mri(process.argv.slice(2));
+const args = mri(process.argv.slice(2), {
+  alias: {
+    'resolve-config': 'resolveConfig',
+  },
+});
```

Comment:
had to add an alias here otherwise when I passed `no-resolve-config` it was only resolved as a variable named `resolve-config` instead of `resolveConfig`